Nature Publishing Group.js

```diff
@@ -269,9 +269,12 @@ function getKeywords(doc) {
 }
 
 //get PDF url
-function getPdfUrl(url) {
+function getPdfUrl(doc, url) {
 	var m = url.match(/(^[^#?]+\/)(?:full|abs)(\/[^#?]+?\.)[a-zA-Z]+(?=$|\?|#)/);
 	if (m && m.length) return m[1] + 'pdf' + m[2] + 'pdf';
+	else if (attr(doc, 'a[data-track-action="download pdf"]', 'href')) {
+		return attr(doc, 'a[data-track-action="download pdf"]', 'href');
```

Comment:
Is the check before here needed? `attr` should return `null` if the element is not found, which should be okay here.